src/esp/gfx/Simulator.cpp

```diff
@@ -57,6 +57,10 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
     houseFilename = cfg.scene.filepaths.at("house");
   }
 
+  if (!io::exists(houseFilename)) {
```

Comment:
You can avoid this by adding a "house" entry to your configuration in  cfg.scene.filepaths (see line 56)

Reply:
Thank you, missed that logic.